### ISSUE TITLE
Empty Select No dropdown on iOs users

### ISSUE THREAD

**Issue by bopnrgi** [yourlabs/django-autocomplete-light#1261]
Hello,

There is a problem with iOS users: on Safari and Chrome. (Test on iPhone, no possibility to test on Mac)

When a user clicks on the field where autocompletion is available, the "classic" list for Select fields is displayed empty. Unable to get the text field where the user must enter his request.

For all users on PC (Ubuntu, Windows) and all browsers the code is functional.
Likewise for the vast majority of Android users, it works.

I have no idea where to look ....

**Comment by OmenApps** [comment]
Experiencing the same with Safari on iPad (7th gen).

Using `autocomplete.ModelSelect2Multiple` in my forms. On any other browser they work as expected. But in Safari, it shows '0 items'.

**Comment by recagonlei** [comment]
I am facing the same problem here. On any other browser, the select workis well, on ios we've notice that happening.
I am using 3.8.2 version

**Comment by OmenApps** [comment]
@aroraumang I'll give that a try this evening. Thanks.

**Comment by belegnar** [comment]
Looks like a kind of minimizer problem
```
[Error] SyntaxError: Invalid regular expression: invalid group specifier name
	(anonymous function) (autocomplete_light.min.js:1)
```

**Comment by oxpeter** [comment]
Thanks @rblack - I had the "SyntaxError" message in Safari's console, and your alteration worked (after I remembered that Django's runserver hosts static files differently when debug=True, and altered the correct js file)

**Comment by leetucker** [comment]
I have been experiencing the same issue, and switching to the latest master helped.  It does look like the issue flagged by @rblack was the culprit.

**Comment by jpic** [comment]
Released in 3.9.0rc2, please reopen if necessary.
